Source/WebKit/UIProcess/RemoteLayerTree/RemoteLayerTreeNode.h

```diff
@@ -124,6 +126,7 @@ class RemoteLayerTreeNode : public CanMakeWeakPtr<RemoteLayerTreeNode> {
     RetainPtr<CALayer> m_layer;
 #if ENABLE(INTERACTION_REGIONS_IN_EVENT_REGION)
     RetainPtr<CALayer> m_interactionRegionsLayer;
+    std::optional<WebCore::FloatRect> m_coverageRect;
```

Comment:
Could we use the 0x0 rectangle as the "invalid value" ? We would save some memory here

Reply:
My comment should have been more complete: if we can use it, we should change this `std::optional<FloatRect>` to a `Markable<FloatRect>` (which keeps the "optional" API and clearly show sthe "optionality" of this member but save the memory)

Reply:
LGTM 👍